### PR TITLE
Adds twitter,packagecloud and CircleCI badges

### PR DESCRIPTION
**Pull request by gurkanindibay**
DESCRIPTION: Adds twitter,packagecloud and CircleCI badges

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -3,6 +3,11 @@
 [![Slack Status](http://slack.citusdata.com/badge.svg)](https://slack.citusdata.com)
 [![Latest Docs](https://img.shields.io/badge/docs-latest-brightgreen.svg)](https://docs.citusdata.com/)
 [![Code Coverage](https://codecov.io/gh/citusdata/citus/branch/master/graph/badge.svg)](https://app.codecov.io/gh/citusdata/citus)
+[![Stack Overflow](https://img.shields.io/badge/Stack%20Overflow-%20-545353?logo=Stack%20Overflow)](https://stackoverflow.com/questions/tagged/citus)
+[![Twitter](https://img.shields.io/twitter/follow/citusdata.svg?label=Follow%20@citusdata)](https://twitter.com/intent/follow?screen_name=citusdata)
+[![Citus Deb Packages](https://img.shields.io/badge/deb-packagecloud.io-844fec.svg)](https://packagecloud.io/app/citusdata/community/search?q=&filter=debs)
+[![Citus Rpm Packages](https://img.shields.io/badge/rpm-packagecloud.io-844fec.svg)](https://packagecloud.io/app/citusdata/community/search?q=&filter=debs)
```
Comment (metdos) [inline README.md:9]: Wrong link

**Comment by metdos** [comment]
1. Let's remove CircleCI until fix the flaky tests. Currently, it will signal that the master branch fails but actually these are flaky tests.
2. Can we reorder badges with new lines? I'm open to other order proposals
docs + stack overflow + slack + code cov
twitter
deb + rpm

File: README.md (hunk 1 above)
Comment (gurkanindibay) [reply README.md]: Fixed

**Comment by gurkanindibay** [comment]
> 1. Let's remove CircleCI until fix the flaky tests. Currently, it will signal that the master branch fails but actually these are flaky tests.
> 2. Can we reorder badges with new lines? I'm open to other order proposals
>    docs + stack overflow + slack + code cov
>    twitter
>    deb + rpm

Removed CircleCI and ordered as you suggested

**Review by metdos — APPROVED** [review]
Added a new line and fixed the slack link. Could you squash and merge

**Comment by metdos** [comment]
@gurkanindibay let's also update the PR description before merging.

**Comment by gurkanindibay** [comment]
> @gurkanindibay let's also update the PR description before merging.

Updated

**Pull request merged**